ydb/docs/en/core/reference/ydb-cli/workload-click-bench.md

```diff
@@ -87,13 +89,9 @@ See the command description to run the load:
 
 ### ClickBench-specific options { #run_clickbench_options }
 
-| Name                                        | Description                                                                                                 | Default value |
-|---------------------------------------------|-------------------------------------------------------------------------------------------------------------|---------------|
-| `--ext-queries <queries>` or `-q <queries>` | External queries to execute during the load, separated by semicolons.                                       |               |
-| `--ext-queries-file <name>`                 | Name of the file containing external queries to execute during the load, separated by semicolons.           |               |
-| `--ext-query-dir <name>`                    | Directory containing external queries for the load. Queries should be in files named `q[0-42].sql`.         |               |
-| `--ext-results-dir <name>`                  | Directory containing external query results for comparison. Results should be in files named `q[0-42].sql`. |               |
-| `--check-canonical` or `-c`                 | Use special deterministic internal queries and compare the results against canonical ones.                  |               |
+| Name               | Description                                              | Default value |
+|--------------------|----------------------------------------------------------|---------------|
+| `--syntax <value>` | Which query syntax option should be used, `yql` or `pg`. | `yql`         |
```

Comment:
The Russian version has a nice comment that `pg` stands for PostgreSQL. Let's add it here too, and also links to https://ydb.tech/docs/en/yql/reference/ and https://ydb.tech/docs/en/postgresql/intro